src/org/usfirst/frc/team166/robot/commands/DriveStraightGyro.java

```diff
@@ -0,0 +1,53 @@
+package org.usfirst.frc.team166.robot.commands;
+
+import edu.wpi.first.wpilibj.Preferences;
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+import org.usfirst.frc.team166.robot.Robot;
+
+/**
+ *
+ */
+public class DriveStraightGyro extends Command {
+	double distance = Preferences.getInstance().getDouble("Distance driven with gyro", 0.0);
+	double power = Preferences.getInstance().getDouble("Speed to drive with gyro", 0.0);
+
+	public DriveStraightGyro(/* double desiredDistance, double desiredPower */) {
+		requires(Robot.drive);
+		// distance = desiredDistance;
+		// power = desiredPower;
+		SmartDashboard.putData("DriveStraightGyro", new DriveStraightGyro());
```

Comment:
This will put a new `DriveStraightGyro` onto the dashboard. And it happens whenever a new `DriveStraightGyro` is created. What you were probably trying to do is to do `SmartDashboard.putData("DriveStraightGyro", this);`. But even then, you probably want to add it to the dashboard from within the `OI` class.